lambdas/service/app.py

```diff
@@ -705,7 +706,7 @@ def validate_catalog(catalog):
         raise BRE(e)
     else:
         if catalog not in config.catalogs:
-            raise BRE(f'Catalog name {catalog!r} is invalid. '
+            raise NFE(f'Catalog name {catalog!r} is invalid. '
```

Comment:
```suggestion
            raise NotFoundError(f'Catalog name {catalog!r} is invalid. '
```

We only aliased `BadRequestError` because it was so prevalent *and* made wrapping awkward in some places.

Reply:
```suggestion
            raise NFE(f'Catalog {catalog!r} does not exist. '
```